test/FastTests/Client/Indexing/IndexExtensionFromClient.cs

```diff
@@ -45,7 +45,7 @@ public void CanCompileIndexWithExtensions(Options options)
             }
         }
 
-        [Fact]
+        [RavenFact(RavenTestCategory.ClientApi | RavenTestCategory.QueryingWithIndexes)]
```

Comment:
wehat is the diff between QueryingWithIndexes and Querying | Indexes ?

Reply:
They are the same, in fact I am leaning against those combined ones and just retire them.